data/events.json

```diff
@@ -0,0 +1,558 @@
+  },
+  {
+    "eventDbId": "6000000082",
+    "studyDbId": "6000000010",
+    "eventDescription": "mechanical harvest for green-chop biomass yield estimates, using Kemper small-plot harvester and customized Carter weigh-wagon",
+    "date": {
+      "startDate": "2017-09-11",
+      "endDate": "2017-09-16"
+    },
+    "eventParameters": [
+        {
+          "code": "HACOM",
```

Comment:
Is this harvest event supposed to be of the same format as the very last one, starting on line 549? The code and name are flipped, and the unit is different.

Reply:
fixed, thanks for finding that!